core/src/tab/tab.stories.tsx

```diff
@@ -8,18 +8,21 @@ const tabs: Tab[] = [
 	{ id: "Second", title: "Second", pane: () => <p>2nd</p> },
 ];
 
-storiesOf("Tab", module).add("Primary", () => (
-	<div>
-		<Tabs children={tabs} />
-		<DivPx size={16} />
-		<Tabs children={tabs} noPadding />
-		<DivPx size={16} />
-		<Tabs style={Tabs.styles.flat} children={tabs} />
-		<DivPx size={16} />
-		<Tabs style={Tabs.styles.flat} children={tabs} noPadding />
-		<DivPx size={16} />
-		<div style={{ height: 200 }}>
-			<Tabs children={tabs} fullHeight />
+storiesOf("Tab", module).add("Primary", () => {
+	const [activeTab, setActiveTab] = useState('Second')
+	return (
```

Comment:
Should not replace the whole story. Should add another one for controlled case, while keeping the uncontrolled case